test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CanonicalSearchTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Linq;
+using System.Web;
+using Hl7.Fhir.Model;
+using Microsoft.Health.Fhir.Client;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Xunit;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.Health.Fhir.Tests.E2E.Rest.Search
+{
+    [HttpIntegrationFixtureArgumentSets(DataStore.All, Format.Json)]
+    public class CanonicalSearchTests : SearchTestsBase<CanonicalSearchTestFixture>
```

Comment:
I just ran these tests on a non-empty cosmos DB database and they all failed. Errors like:
```
  Message: 
    Assert.Collection() Failure
    Collection: [EntryComponent { Base = null, Children = [...], ElementId = null, Extension = [...], FullUrl = "http://localhost/Observation/c95003cc-ce0c-44f3-bb"..., ... }, EntryComponent { Base = null, Children = [...], ElementId = null, Extension = [...], FullUrl = "http://localhost/Observation/1e867f7d-6aa9-46af-bd"..., ... }, EntryComponent { Base = null, Children = [...], ElementId = null, Extension = [...], FullUrl = "http://localhost/Observation/17d45a72-2eeb-45cb-a6"..., ... }, EntryComponent { Base = null, Children = [...], ElementId = null, Extension = [...], FullUrl = "http://localhost/Observation/d087cbd4-6162-47ef-b4"..., ... }, EntryComponent { Base = null, Children = [...], ElementId = null, Extension = [...], FullUrl = "http://localhost/Observation/79fb0b4e-a790-4222-ac"..., ... }, ...]
    Expected item count: 1
    Actual item count:   10
  Stack Trace: 
    CanonicalSearchTests.GivenAnObservationWithProfile_WhenSearchingByCanonicalUriVersion_Then1ExpectedResultIsFound() line 61
    --- End of stack trace from previous location where exception was thrown ---
```

I think this is because _profile is not considered supported. What is the guidance here?

Reply:
We would need to reindex. These results look like the _profile search was probably dropped and returned more items than expected

Reply:
I could potentially inspect the capability statement to ensure _profile can be searched for these tests, else skip